### ISSUE TITLE
 every time I breed horses my minecraft crashes

### ISSUE THREAD

**Issue by VinicinDuMau** [SanityCh3ck/horse-stonks#11]
When I breed horses, before they can even generate offspring, Minecraft crashes. I don't know what it could be, and since I haven't seen any other comments complaining about the bug, I don't know if it's just me.

**Comment by D10King** [comment]
experiencing the same issue using the fabric loader in MC 1.21.1

**Comment by Ponoit** [comment]
I have the same problem did you guys find a solution

**Comment by Russeki** [comment]
> I have the same problem did you guys find a solution

For now there's no solution. I had to uninstall the mod just to open my world.

**Comment by lookstothemoon** [comment]
+1

**Comment by ergvt** [comment]
me too!

**Comment by PoisonLoca** [comment]
Yeah, it happened to me while was on fabric. doesn't look like it will be fixed. I was going to post this exact problem. It's a neat mod just a shame we cannot use it.
